### PR TITLE
fix/healthy_screen

### LINKED ISSUE #98: Back arrow not responding when i click on it and the text color change to black with wrong position with scrolling , please fix this issue
image
image

### PR DESCRIPTION
**Pull request by Abanoub-Nagy**

### CODE REVIEW

File: app/src/main/res/drawable/linear_collapsing_overlay.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android">
+    <gradient
+        android:angle="-90"
+        android:centerColor="#00000000"
```
Comment (AmeerAmjed) [inline app/src/main/res/drawable/linear_collapsing_overlay.xml]: why hardcode color

**Review by mosayed01 — APPROVED** [review]

File: app/src/main/res/layout/fragment_healthy.xml [hunk 2]
```diff
@@ -51,9 +54,9 @@
                 android:layout_height="?attr/actionBarSize"
                 android:background="@android:color/transparent"
                 android:minHeight="?attr/actionBarSize"
-                android:paddingBottom="@dimen/spacing_large"
                 app:layout_collapseMode="pin"
                 app:navigationIcon="@drawable/icon_back"
+                app:navigationIconTint="@color/black"
                 app:title="Avoid ultra-processed food" />
```
Comment (AmeerAmjed) [inline app/src/main/res/layout/fragment_healthy.xml]: why title hardcode

Comment (mosayed01) [inline app/src/main/res/layout/fragment_healthy.xml]: hardcode

File: app/src/main/res/layout/fragment_healthy.xml [hunk 3]
```diff
@@ -17,32 +17,35 @@
...
+            app:expandedTitleTextColor="@color/tertiary_60"
+            app:layout_collapseMode="parallax"
+            app:layout_collapseParallaxMultiplier="0.5"
             app:layout_scrollFlags="scroll|exitUntilCollapsed"
             app:maxLines="1"
             app:titleTextEllipsize="end">
 
-            <ImageView
+            <com.google.android.material.imageview.ShapeableImageView
                 android:id="@+id/healthy_image"
```
Comment (AmeerAmjed) [inline app/src/main/res/layout/fragment_healthy.xml]: rename id to **image_view_healthy**

File: app/src/main/res/layout/fragment_healthy.xml [hunk 4]
```diff
@@ -51,9 +54,9 @@
                 android:layout_height="?attr/actionBarSize"
                 android:background="@android:color/transparent"
                 android:minHeight="?attr/actionBarSize"
-                android:paddingBottom="@dimen/spacing_large"
                 app:layout_collapseMode="pin"
                 app:navigationIcon="@drawable/icon_back"
+                app:navigationIconTint="@color/black"
```
Comment (mosayed01) [inline app/src/main/res/layout/fragment_healthy.xml]: use color tertiary

**Review by mosayed01 — CHANGES_REQUESTED** [review]

File: app/src/main/res/drawable/linear_collapsing_overlay.xml (hunk 1 above)
Comment (AmeerAmjed) [reply app/src/main/res/drawable/linear_collapsing_overlay.xml]: use @android:color/transparent

**Review by mosayed01 — DISMISSED** [review]

**Pull request merged**